src/pages/LandingPage.py

```diff
@@ -15,6 +15,7 @@ class LandingPage(BaseWrapper):
     LOG_OUT_BTN_CSS = "div.text-right > div"
     All_PAGE_CSS = "html"
     UPCOMING_PUBLIC_EVENT = "img[alt~='Event']"
+    JOIN_EVENT_CSS = ".events-article>div>div:last-of-type>a"
```

Comment:
It directs to 'explore more events'.
Regarding 'join event', each public event at the landing page will contain 'join event', not just one. You can find them by '//a[@class='link']'.